projects/observability/src/shared/components/gauge/gauge.component.ts

```diff
@@ -22,12 +18,12 @@ import { Arc, arc, DefaultArcObject } from 'd3-shape';
               class="value-ring"
               *ngIf="rendererData.radius > ${GaugeComponent.GAUGE_MIN_RADIUS_TO_SHOW_PATH}"
               [attr.d]="rendererData.data.valueArc"
-              [attr.fill]="rendererData.data.threshold.color"
+              [attr.fill]="rendererData.data.color"
             />
-            <text x="0" y="-4" class="value-display" [attr.fill]="rendererData.data.threshold.color">
+            <text x="0" y="0" class="value-display" [attr.fill]="rendererData.data.color">
```

Comment:
Other than the changes in the render data access, were the changes to y values and the radius threshold above intended?

Reply:
Already rolled back